pkg/verify/verify.go

```diff
@@ -30,6 +30,17 @@ type Interface interface {
 	Verify(ctx context.Context, releaseDigest string) error
 }
 
+// SignatureStore retrieves signatures for a digest or returns an error. It requires String()
+// for describing the store.
```

Comment:
nit: I'd drop the `It requires String...` sentence, since the `String` method is motivated below inside the `interface` block.  `SignatureStore is a store from which signatures can be retrieved.` or something is probably a sufficient comment for the interface as a whole.  Or maybe `SignatureStore is a store selling signatures.` or some such ;).